server/node/README.md

```diff
@@ -7,3 +7,76 @@ Start the server by running:
 ```bash
 npm start
 ```
+
+---
+
+### `GET /paypal-api/auth/browser-safe-client-token`
+
+**Description:**  
+Generates and returns a browser-safe client token for initializing the PayPal JavaScript SDK on the client side.
+
+**Responses:**
+
+- `200 OK`  
+  Returns a JSON object with the client token.
+
+  ```json
+  {
+    "access_token": "abc123...",
+    "expires_in": "900" // TTL in seconds,
+    // additional metadata
+  }
+  ```
+  For more details, visit PayPal's [Developer Documentation](https://developer.paypal.com/api/rest/authentication/) site 
+
+
+---
+
+### `POST /paypal-api/checkout/orders/create`
+
+**Description:**  
+Creates a new PayPal order with sample data. This simulates the beginning of a checkout session.
+
+**Request Body:**
+No body required.
```

Comment:
Could we add a comment here mentioning that the server creates an order with sample order data? I think technically the pattern is that the client will send the server the order payload but if we're making an exception here maybe some comment could help clarify?